charm4py/charm.py

```diff
@@ -541,18 +541,14 @@ def registerMainModule(self):
             import platform
             from . import charm4py_version
             py_impl = platform.python_implementation()
-            out_msg = ("Charm4py> Running Charm4py version " + charm4py_version +
+            print("Charm4py> Running Charm4py version " + charm4py_version +
                        " on Python " + str(platform.python_version()) + " (" +
                        py_impl + "). Using '" +
                        self.lib.name + "' interface to access Charm++")
             if py_impl == 'PyPy':
```

Comment:
I think you should change this line to be:
```suggestion
            if py_impl != 'Cython':
```

so it covers all possible Python implementations.